{
  "_class": "hudson.tasks.junit.TestResult",
  "testActions": [],
  "duration": 0.01,
  "empty": false,
  "failCount": 3,
  "passCount": 3,
  "skipCount": 0,
  "suites": [
    {
      "cases": [
        {
          "testActions": [],
          "age": 0,
          "className": "analyze(null]; should throw TypeError",
          "duration": 0.003,
          "errorDetails": null,
          "errorStackTrace": null,
          "failedSince": 0,
          "name": "exceptions analyze(null]; should throw TypeError",
          "skipped": false,
          "skippedMessage": null,
          "status": "PASSED",
          "stderr": null,
          "stdout": null
        },
        {
          "testActions": [],
          "age": 0,
          "className": "analyze([]); should throw Error",
          "duration": 0.001,
          "errorDetails": null,
          "errorStackTrace": null,
          "failedSince": 0,
          "name": "exceptions analyze([]); should throw Error",
          "skipped": false,
          "skippedMessage": null,
          "status": "PASSED",
          "stderr": null,
          "stdout": null
        }
      ],
      "duration": 0.004,
      "enclosingBlockNames": [],
      "enclosingBlocks": [],
      "id": null,
      "name": "exceptions",
      "nodeId": null,
      "stderr": null,
      "stdout": null,
      "timestamp": "2018-01-08T14:58:27"
    },
    {
      "cases": [
        {
          "testActions": [],
          "age": 0,
          "className": "analyze([4, 2, 6, 1, 3, 7, 5, 3]; should not modify the argument",
          "duration": 0.002,
          "errorDetails": null,
          "errorStackTrace": null,
          "failedSince": 0,
          "name": "arguments analyze([4, 2, 6, 1, 3, 7, 5, 3]; should not modify the argument",
          "skipped": false,
          "skippedMessage": null,
          "status": "PASSED",
          "stderr": null,
          "stdout": null
        }
      ],
      "duration": 0.002,
      "enclosingBlockNames": [],
      "enclosingBlocks": [],
      "id": null,
      "name": "arguments",
      "nodeId": null,
      "stderr": null,
      "stdout": null,
      "timestamp": "2018-01-08T14:58:27"
    },
    {
      "cases": [
        {
          "testActions": [],
          "age": 15,
          "className": "analyze([4, 2, 6, 1, 3, 7, 5, 3]; should return {{max: 7, mean: 3.875, median: 3.5, min: 1, mode: [3], range: 6}}",
          "duration": 0.003,
          "errorDetails": null,
          "errorStackTrace": "AssertionError: expected { Object (max, mean, ...) } to deeply equal { Object (max, mean, ...) }\n    at Context.<anonymous> (test/statistics.test.js:39:65)",
          "failedSince": 1,
          "name": "return values analyze([4, 2, 6, 1, 3, 7, 5, 3]; should return {{max: 7, mean: 3.875, median: 3.5, min: 1, mode: [3], range: 6}}",
          "skipped": false,
          "skippedMessage": null,
          "status": "FAILED",
          "stderr": null,
          "stdout": null
        },
        {
          "testActions": [],
          "age": 15,
          "className": "analyze([3, 5, 2, -5, 9, 2, -5, 5, 10, 4, 1, 0, -1, 9, 0]; should return {{max: 10, mean: 2.6, median: 2, min: -5, mode: [-5, 0, 2, 5, 9], range: 15}}",
          "duration": 0.001,
          "errorDetails": null,
          "errorStackTrace": "AssertionError: expected { Object (max, mean, ...) } to deeply equal { Object (max, mean, ...) }\n    at Context.<anonymous> (test/statistics.test.js:44:90)",
          "failedSince": 1,
          "name": "return values analyze([3, 5, 2, -5, 9, 2, -5, 5, 10, 4, 1, 0, -1, 9, 0]; should return {{max: 10, mean: 2.6, median: 2, min: -5, mode: [-5, 0, 2, 5, 9], range: 15}}",
          "skipped": false,
          "skippedMessage": null,
          "status": "FAILED",
          "stderr": null,
          "stdout": null
        },
        {
          "testActions": [],
          "age": 15,
          "className": "analyze([5, 1, 1, 1, 3, -2, 2, 5, 7, 4, 5, 16]); should return {max: 16, mean: 4, median: 3.5, min: -2, mode: [1, 5], range: 18}",
          "duration": 0,
          "errorDetails": null,
          "errorStackTrace": "AssertionError: expected { Object (max, mean, ...) } to deeply equal { Object (max, mean, ...) }\n    at Context.<anonymous> (test/statistics.test.js:49:79)",
          "failedSince": 1,
          "name": "return values analyze([5, 1, 1, 1, 3, -2, 2, 5, 7, 4, 5, 16]); should return {max: 16, mean: 4, median: 3.5, min: -2, mode: [1, 5], range: 18}",
          "skipped": false,
          "skippedMessage": null,
          "status": "FAILED",
          "stderr": null,
          "stdout": null
        }
      ],
      "duration": 0.004,
      "enclosingBlockNames": [],
      "enclosingBlocks": [],
      "id": null,
      "name": "return values",
      "nodeId": null,
      "stderr": null,
      "stdout": null,
      "timestamp": "2018-01-08T14:58:27"
    }
  ]
}
